package.json

```diff
@@ -23,14 +23,17 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "npm run lint && mocha --recursive tests/*.spec.js -R spec && node tests/e2e.js",
+    "test": "npm run lint && npm run test-node && npm run test-electron",
+    "test-node": "mocha --recursive tests/*.spec.js -R spec && node tests/e2e.js",
+    "test-electron": "electron-mocha --recursive tests/*.spec.js -R spec",
```

Comment:
I ported them already (just pushed), but discovered quite a few of those tests are actually broken (but passed regardless of outcome) – also fixed them, but the bmaps test case will need an assert or two more.